toolkit/sampler.py

```
@@ -200,6 +199,8 @@ def get_sampler(
     pipe = StableDiffusionKDiffusionPipeline.from_pretrained("stabilityai/stable-diffusion-2-1-base")
     pipe = pipe.to("cuda")
 
+    from k_diffusion.external import CompVisDenoiser
+
     k_diffusion_model = CompVisDenoiser(model)
```

Ruff (0.15.15)

[error] Undefined name `model`

(F821)